repose-aggregator/src/docs/asciidoc/recipes/role-based-access-control.adoc

```diff
@@ -41,13 +41,10 @@ Please refer to the <<../filters/keystone-v2.adoc#,Keystone v2 filter>> document
 
 == 3. Configure the RBAC filter
 
-There are two mechanisms for doing the authorization side.
-If your API is minimal or your just getting started with it, then you might find the Simple RBAC filter most useful.
+There are two mechanisms for doing the Authorization side.
+If your API is minimal or your just getting started with it, then you might find the <<../filters/simple-rbac.adoc#,Simple RBAC filter>> most useful.
 It uses a very simple Domain Specific Language (DSL) that is similar to what other tools use for this basic mechanism.
-If on the other had your API is large and/or your authorizations are complex, then you will need the heavy lifting of the API Validation filter.
-Since the Simple way is very self explanatory, we will only dive into the more complex way here.
-
-You will do most of your RBAC configuration here.
+If on the other had your API is large and/or your authorizations are complex, then you will need the heavy lifting of the <<../filters/api-validator.adoc#,API Validator filter>>.
 
 === 1. What do you need?
```

Comment:
Not that you made changes to it, but we should make a note to look at this type of headering, because the numbering isn't very clear when i look at the generated doc.

Reply:
Here is a link to the last set of published docs for that reference:
* http://www.openrepose.org/versions/8.6.3.0/recipes/role-based-access-control.html#_1_what_do_you_need